app/src/main/res/values/arrays.xml

```diff
@@ -36,4 +36,13 @@
         <item>DARK</item>
     </string-array>
 
+    <string-array name="pref_entries_background_type_select" translatable="false">
+        <item>@string/background_type_map</item>
+        <item>@string/background_type_aerial_esri</item>
+    </string-array>
+
+    <string-array name="pref_entryvalues_background_type_select" translatable="false">
+        <item>MAP</item>
+        <item>AERIAL_ESRI</item>
```

Comment:
Why does the option name include "Esri" here? If at some point in the future the tile provider is changed, the option saved in the database will be outdated/wrong.

Reply:
Good question, I assumed that in such case it would mean rewrite to multiple aerials, but either single provider can be exchanged for another or SC will autoselect best imagery.

Reply:
Changed, thanks!